### ISSUE TITLE
Privacy Policy

### ISSUE THREAD

**Issue by the-thumb** [Bungie-net/api#1002]
TLDR: Is there a location where it states how user information can be stored and shared?

I am looking to make an app that will combine the progress of members of a fireteam. I feel like I might be in a grey area here... When I prompt the user for authorization It will be made clear that their progress will be exposed to other users and for what purpose. If this is outright not allowed, I would like to stop working on this idea. =P If there are specifics in the policies on how it is required to be complaint, that would be useful. If there are no set rules, I am open to opinion and would like to proceed with caution.

**Comment by ckhicks** [comment]
There's quite a bit listed here: https://www.bungie.net/7/en/Legal/PrivacyPolicy

**Comment by as-com** [comment]
Regardless, you should always have both a privacy policy _and_ make it clear to your users what information will be public.

**Comment by the-thumb** [comment]
Groovy. So there is no specific documention in regards to the API or "Apps"? Thank you @as-com, it had not crossed my mind yet to include a proper privacy policy on the site/app I am creating.

**Comment by vthornheart-bng** [comment]
It sounds to me like this is a legitimate use case.  I definitely think it'll be important to give your users a heads up as to what you're doing with their data but I have no objections to it!

You will want to check up on GDPR and see if it applies to you/what you might want or need to do to allow people to opt-out and purge their data later, but that's a separate issue from our own terms and conditions.

**Comment by the-thumb** [comment]
@vthornheart-bng Groovy! I'll add a task for checking on GDPR compliance before anything is made public.
